internal/store/persistentvolume.go

```diff
@@ -32,11 +32,49 @@ import (
 )
 
 var (
+	descPersistentVolumeClaimRefName          = "kube_persistentvolume_claimref_info"
+	descPersistentVolumeClaimRefHelp          = "PersistentVolume spec.claimRef converted to Prometheus labels."
+	descPersistentVolumeClaimRefDefaultLabels = []string{"persistentvolume"}
+
 	descPersistentVolumeLabelsName          = "kube_persistentvolume_labels"
 	descPersistentVolumeLabelsHelp          = "Kubernetes labels converted to Prometheus labels."
 	descPersistentVolumeLabelsDefaultLabels = []string{"persistentvolume"}
 
 	persistentVolumeMetricFamilies = []generator.FamilyGenerator{
+		*generator.NewFamilyGenerator(
+			descPersistentVolumeClaimRefName,
+			descPersistentVolumeClaimRefHelp,
+			metric.Gauge,
+			"",
+			wrapPersistentVolumeFunc(func(p *v1.PersistentVolume) *metric.Family {
+				claimRef := p.Spec.ClaimRef
+
+				if claimRef == nil {
+					return &metric.Family{
+						Metrics: []*metric.Metric{},
+					}
+				}
+				return &metric.Family{
+					Metrics: []*metric.Metric{
+						{
+							LabelKeys: []string{
+								"apiversion",
+								"kind",
+								"name",
+								"namespace",
```

Comment:
I would call this `claim_namespace`, but open to other suggestions, otherwise would be confusing if this is the namespace of the PV or the PVC.